<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="ui_splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="ui_twExtTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="ui_widSummaryTab">
        <attribute name="title">
         <string>Summary</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="ui_pteSummary">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ui_widExtDataTab">
        <attribute name="title">
         <string>Extended Data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="ui_pteExtData">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QsciScintilla" name="ui_contentEditor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="ui_widFind" native="true">
      <layout class="QHBoxLayout" name="ui_findLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="ui_leFindText">
         <property name="placeholderText">
          <string>Find text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_pbFind">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Find</string>
         </property>
         <property name="shortcut">
          <string>F3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_pbFindBackwards">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Find backwards</string>
         </property>
         <property name="shortcut">
          <string>Shift+F3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_leReplaceText">
         <property name="placeholderText">
          <string>Replace Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_pbReplace">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Replace</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+=</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="ui_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_copy"/>
    <addaction name="action_cut"/>
    <addaction name="action_paste"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QStatusBar" name="ui_statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="ui_toolBar">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_copy"/>
   <addaction name="action_cut"/>
   <addaction name="action_paste"/>
  </widget>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../BGCMS/bgcms.qrc">
     <normaloff>:/imgs/save.png</normaloff>:/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="../../BGCMS/bgcms.qrc">
     <normaloff>:/imgs/copy.png</normaloff>:/imgs/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="../../BGCMS/bgcms.qrc">
     <normaloff>:/imgs/cut.png</normaloff>:/imgs/cut.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="../../BGCMS/bgcms.qrc">
     <normaloff>:/imgs/paste.png</normaloff>:/imgs/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="../../BGCMS/bgcms.qrc">
     <normaloff>:/imgs/undo.png</normaloff>:/imgs/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="../../BGCMS/bgcms.qrc">
     <normaloff>:/imgs/redo.png</normaloff>:/imgs/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
   <container>1</container>
   <slots>
    <signal>copyAvailable(bool)</signal>
    <slot>copy()</slot>
    <slot>cut()</slot>
    <slot>paste()</slot>
    <slot>undo()</slot>
    <slot>redo()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../BGCMS/bgcms.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_save</sender>
   <signal>triggered()</signal>
   <receiver>EditorWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_contentEditor</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_copy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_copy</sender>
   <signal>triggered()</signal>
   <receiver>ui_contentEditor</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_contentEditor</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_cut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_cut</sender>
   <signal>triggered()</signal>
   <receiver>ui_contentEditor</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_paste</sender>
   <signal>triggered()</signal>
   <receiver>ui_contentEditor</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_close</sender>
   <signal>triggered()</signal>
   <receiver>EditorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_undo</sender>
   <signal>triggered()</signal>
   <receiver>ui_contentEditor</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_redo</sender>
   <signal>triggered()</signal>
   <receiver>ui_contentEditor</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbFind</sender>
   <signal>clicked()</signal>
   <receiver>EditorWindow</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>823</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbFindBackwards</sender>
   <signal>clicked()</signal>
   <receiver>EditorWindow</receiver>
   <slot>findBackwards()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbReplace</sender>
   <signal>clicked()</signal>
   <receiver>EditorWindow</receiver>
   <slot>replace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>find()</slot>
  <slot>findBackwards()</slot>
  <slot>replace()</slot>
 </slots>
</ui>
